composer.json

```diff
@@ -60,7 +60,6 @@
         "patches": {
             "drupal/core": {
                 "Users can't reference unpublished content even when they have access to it. See https://www.drupal.org/project/drupal/issues/2845144": "https://www.drupal.org/files/issues/2024-02-13/2845144-87.patch",
-                "Content moderation and Workspaces https://www.drupal.org/project/drupal/issues/3179199#comment-15711680": "https://www.drupal.org/files/issues/2024-08-11/3179199-3132022-content-moderation-workspaces-query.patch"
```

Comment:
Hi @finnlewis with this line removed, the line above needs the trailing comma removed

Reply:
oooh - thanks dude!